paginate-nfts/README.md

```diff
@@ -0,0 +1,96 @@
+# Paginating NFTs for a given wallet address
+
+Pagination is not really a thing when it comes to the `getProgramAcccount` RPC method. This tutorial demonstrates how you can still simulate NFT pagination by lazy loading NFTs and fetching their metadata on demand.
+
+In this tutorial, we are going to:
+- Use the `findAllByOwner` method.
+- Store the list of lazy-loaded NFTs.
+- Fetch the metadata of for each NFT in the current page.
+
+![image](https://user-images.githubusercontent.com/56197821/176737427-43df0aca-6ea9-443f-b9e9-718bdb654ab4.png)
+
+Okay, let's get started! 🔥
+
+Note that you can find all the code below in this [`pages/index.js` file](./pages/index.js).
+
+1. **Create a new Next.js app.**
+   
+   We will be using the [findAllByOwner](https://github.com/metaplex-foundation/js#findallbyowner) method in the Metaplex SDK which will give us the list of NFTs without their JSON metadata loaded. This is important because if we had to fetch hundreds of NFTs with their JSON metadata, it will take a lot of time and greatly impact the performance of the application.
```

Comment:
This sentence should have been left in the intro. It does not affect the Next.js step. Please move it back between the first and the second paragraph of the into.